play.py

```diff
@@ -0,0 +1,148 @@
+import importlib
+import time
+import random #in a module, where are all packages imported
+
+# import word category when specified by the user
+# user chooses to play fruits, animal or places or random_words
+def chooseCategory(dict_name):
+    try:
+        module = importlib.import_module("dictionaries")
+
+        # create instance of class
+        dicts = module.Dictionaries()
+
+        # get reference to method and call the method
+        dict_method = getattr(dicts, dict_name)
+        chosenCategory = dict_method()
```

Comment:
If change the way dictionaries are initialised....

```suggestion
        chosenCategory = getattr(dicts, dict_name)
```

Reply:
I am unable to use the `getattr()` because the dict_name variable is a string returns a string

Reply:
I am unable to use the `getattr()` because the dict_name variable is a string. I am using `vars()` to return all variables in the class